### PR TITLE
Feat/adding selectable options

### PR DESCRIPTION
**Pull request by sergio9977**
Reference: https://linear.app/subscline/issue/SUB-875/adding-selectable-options-for-number-of-displayed-items

image

- Store was added
- Datatable was updated

### CODE REVIEW

**Review by JangYoonsung — COMMENTED** [review]
The goal is not to remove existing features and add display options, but to retain the existing features and add display options.

**Comment by sergio9977** [comment]
hi @JangYoonsung I did it like this because in the desired result that is in the ticket there are no options to go between pages, but I will fix it and I will add that functionality again
image

File: DataTable.svelte
Comment (JangYoonsung) [inline DataTable.svelte]:
```suggestion
      <Select
        values={options}
        selected={pageSize.toString()}
        reverse
        onChangeSelected={handleChangePagination}
      />
```
If reverse is always true, you can write like this

File: DataTable.svelte
Comment (suzukey) [inline DataTable.svelte]: nits: It's hard to know what the options are, so let's be explicit.
e.g. `options` -> `pageSizeOptions`

File: DataTable.svelte
Comment (suzukey) [inline DataTable.svelte]: Why does it jump to the last page?
Is there a reason?

I thought the first page might be good.

**Review by suzukey — COMMENTED** [review]
thank you!
It's pretty close to completion!
I think I can approve if you reply or fix the comment.

File: DataTable.svelte
Comment (sergio9977) [reply DataTable.svelte]: yes you are right I just changed that

Comment (suzukey) [reply DataTable.svelte]: Thank you for the change!
Sorry, I'm on vacation until May 8th.
Please wait for another person's review

**Review by JangYoonsung — APPROVED** [review]
lgtm

**Pull request merged**
